Assets/MirageXR/Tests/NewUI/RootView_v2.cs

```diff
@@ -125,19 +122,19 @@ private void OnStepsClick(bool value)
         if (value) _pageView.currentPageIndex = 0;
     }
 
-    private void OnViewClick(bool value)
+    private void OnViewClick()
     {
-        if (value) _pageView.currentPageIndex = 1;
+        PopupsViewer.Instance.Show(_profilePrefab);
     }
```

Comment:
change `onViewClick` to `OnProfileClick` (or `OnProfileClicked`)

Reply:
done